### PR TITLE
added maybe page /Feature/FFTF-1174

### PR DESCRIPTION
**Pull request by ellietms**

### CODE REVIEW

File: app/routes/index.js [hunk 1]
```diff
@@ -93,6 +93,15 @@ function createModelNotEligible (backUrl, ineligibleContent) {
   }
 }
 
+const MAYBE_ELIGIBLE = (backUrl, nextPath, maybeEligibleContent) => {
```
Comment (diya2214) [inline app/routes/index.js:96]: Capitals should not be used in function name

Comment (ellietms) [reply app/routes/index.js:96]: sure , it was in water project

Comment (ogbk) [reply app/routes/index.js]: in water ```MAYBE_ELIGIBLE``` was a constant, not a function

**Review by ogbk — APPROVED** [review]
neat code

**Pull request merged**